apps/login-test-acceptance/tests/user.ts

```diff
@@ -124,11 +120,8 @@ export class PasswordUserWithTOTP extends User {
 
   async ensure(page: Page) {
     await super.ensure(page);
-
     this.secret = await addTOTP(this.getUserId());
-
-    // wait for projection of user
-    await page.waitForTimeout(2000);
+    await eventualNewUser(this.getUserId());
```

Comment:
Why do you wait for the already created user here and not for the created TOTP?

Reply:
Yes, that would be better and we should do that (if the API allows it) when we fix the acceptance tests.